### ISSUE TITLE
sdpTransform only seems to affect the return stream

### ISSUE THREAD

**Issue by Ziggy2001** [peers/peerjs#756]
Got a puzzler here and I'm sure I'm missing something simple. I am trying to adjust the bandwidth of the peerjs call and whilst it works on the stream coming back to me from the server, my stream going to the server doesn't appear to be affected.

So I've got this on the server which is waiting for calls:

```
    peer.on('call', (call)=>{
        call.answer(stream,{sdpTransform:(sdp)=>{
            sdp = modifySDP(sdp);
            console.log(sdp);
            return sdp;
        }});
        call.on('stream', (remoteStream)=>{
            console.log("REMOTESTREAM");
            playStream(remoteStream);
        });           
    });
```

The modifySDP routine is a function which does just that. It modifies the SDP by simply taking the SDP and either amending or adding b=AS:8

I'm using 8 as the bitrate because I can tell for sure if it's working as it sounds like a bad telephone.

On the client side I'm using:

```
        let call = peer.call(callID, _microphone, {sdpTransform:(sdp)=>{
            sdp = modifySDP(sdp);
            console.log(sdp);
            return sdp;
        }});

        call.on('stream', (remoteStream)=>{
            console.log("REMOTESTREAM");
            playStream(remoteStream);
        });
```

What I have noticed is on the server the REMOTESTREAM in the on stream event gets logged before the console.log(sdp) in the call.answer which makes me think the answer hasn't fully resolved before the stream is received.

So to sum up, using this, the client hears the audio from the server in 8 bit, but the server is getting the client audio unmodified not in 8 bit.

Any ideas how to resolve this so both client and server are in 8 bit?
